schemas/ingredient.json

```diff
@@ -0,0 +1,72 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "http://ns.adobe.com/c2patool/ingredient-definition/v1",
+	"type": "object",
+	"description": "Definition format for an ingredient created with c2patool",
+	"examples": [
+		{  
+            "title": "C.jpg",
+            "format": "image/jpeg",
+            "instance_id": "xmp:iid:01712cdc-26b8-4902-ad53-1cdbd5370b1b",
+            "relationship": "componentOf",
+            "thumbnail": {
+                "format": "image/jpeg",
+                "identifier": "thumb.jpg"
+            },
+            "active_manifest": "manifest.c2pa"
+		}
+    ],
```

Comment:
Nit: there's some formatting funkiness here. You can use https://jsonformatter.org/json-pretty-print to format it or maybe there's a VS Code plugin?

Reply:
it had a combination of tabs and spaces, looked ok on my machine, updated now.

Reply:
replaced all tabs with spaces